### PR TITLE
uh oh we sold the station

### PR DESCRIPTION
**Pull request by Gef21**
whoops

### CODE REVIEW

**Comment by Megamaw** [comment]
i approve

**Comment by Exxion** [comment]
Sorry to be a killjoy but I think we've all had our fun here

File: code/game/jobs/job/captain.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	minimal_access = list() 	//See get_access()
 	minimal_player_age = 14
```
Comment (Iamgoofball) [inline code/game/jobs/job/captain.dm:14]: This explains a lot about your players

**Comment by Intigracy** [comment]
Repoban goofball when

**Comment by ghost** [comment]
I'm all for banning people who come from TG, Intigracy.

**Comment by Iamgoofball** [comment]
to be fair you could say the same about tg's players

**Pull request closed without merging**